src/Umbraco.Core/Services/ServiceContext.cs

```diff
@@ -150,49 +148,30 @@ public class ServiceContext
             if (redirectUrlService != null) _redirectUrlService = new Lazy<IRedirectUrlService>(() => redirectUrlService);
         }
 
-        // fixme - but this is obsoleting a new method?!!?!!?
-        [Obsolete("The IPublishingStrategy parameter is no longer required and not used")]
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public ServiceContext(
-            RepositoryFactory repositoryFactory,
-            IScopeUnitOfWorkProvider dbUnitOfWorkProvider,
-            IScopeUnitOfWorkProvider fileUnitOfWorkProvider,
-            IPublishingStrategy publishingStrategy,
-            CacheHelper cache,
-            ILogger logger,
-            IEventMessagesFactory eventMessagesFactory)
-            : this(repositoryFactory, dbUnitOfWorkProvider, fileUnitOfWorkProvider, cache, logger, eventMessagesFactory)
-        { }
-
-        // fixme CLEAN THIS
-
         /// <summary>
         /// Creates a service context with a RepositoryFactory which is used to construct Services
         /// </summary>
         /// <param name="repositoryFactory"></param>
-        /// <param name="dbUnitOfWorkProvider"></param>
-        /// <param name="fileUnitOfWorkProvider"></param>
+        /// <param name="provider"></param>
         /// <param name="cache"></param>
         /// <param name="logger"></param>
         /// <param name="eventMessagesFactory"></param>
         public ServiceContext(
             RepositoryFactory repositoryFactory,
-            IScopeUnitOfWorkProvider dbUnitOfWorkProvider,
-            IScopeUnitOfWorkProvider fileUnitOfWorkProvider, 
-            CacheHelper cache, 
+            IScopeUnitOfWorkProvider provider,

```

Comment:
Not sure we can avoid breaking things, really, but that's one of the things I want Shan to look at.